packages/dotnet-sdk/src/TeamsFx/wwwroot/jsInterop.js

```diff
@@ -1,23 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { TeamsUserCredential, setLogFunction } from "./teamsfx.js"
-
-// create class instance
-export function createTeamsUserCredential() {
-    return new TeamsUserCredential();
-}
-
-// call .NET function
-export function setLogFunctionCallback(logFunctionCallbackRef) {
-    setLogFunction((level, message) => {
-        logFunctionCallbackRef.invokeMethodAsync('Invoke', level, message);
-    });
-}
-
-export function clearLogFunctionCallback() {
-    setLogFunction(undefined);
-}
 
 export function initialize() {
```

Comment:
In this func the object microsoftTeams is using the model in template? I am not very familiar with this part.